selfdrive/car/toyota/interface.py

```diff
@@ -187,9 +187,10 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[], experi
     ret.enableBsm = 0x3F6 in fingerprint[0] and candidate in TSS2_CAR
     # Detect smartDSU, which intercepts ACC_CMD from the DSU allowing openpilot to send it
     smartDsu = 0x2FF in fingerprint[0]
-    # In TSS2 cars the camera does long control
-    found_ecus = [fw.ecu for fw in car_fw]
-    ret.enableDsu = (len(found_ecus) > 0) and (Ecu.dsu not in found_ecus) and (candidate not in NO_DSU_CAR) and (not smartDsu)
+    if candidate not in (CAR.LEXUS_IS, CAR.LEXUS_RC):
+      # In TSS2 cars the camera does long control
+      found_ecus = [fw.ecu for fw in car_fw]
+      ret.enableDsu = (len(found_ecus) > 0) and (Ecu.dsu not in found_ecus) and (candidate not in NO_DSU_CAR) and (not smartDsu)
```

Comment:
@adeebshihadeh do you remember why your port of the IS was lateral only? Just didn't gather the required static messages we need to send? This list is duplicated in a few places now so I'd like to make a constant